### ISSUE TITLE
[Quest] Boiling Point @ Borean Tundra

### ISSUE THREAD

**Issue by Ic4r0** [WoWManiaUK/Redemption#5168]
**Links:**
https://www.wowhead.com/quest=11627/boiling-point
https://www.wowhead.com/npc=25376/imperean (gives the quest)
https://www.wowhead.com/npc=25416/simmer (target)
https://www.wowhead.com/npc=25418/churn (target)

https://www.youtube.com/watch?v=muNuag2AdM0&feature=youtu.be&ab_channel=WorldOfWarcraftGuidesAndGameplay (walktrough)

**What is Happening:**
When Simmer (or Churn) become friendly it does not "submit". Everything is the same as in the video (they become friendly at a certain point and their dialogues are correct)
*What has been tried:*
- fight them multiple times (each time they became friendly but neither of them "submitted")
- abandon the quest and repeat it (same result)

**What Should happen:**
As can be seen in the youtube video in the links, when you bring Simmer/Churn to low health, it becomes friendly (giving a speech to how they would accept to submit to the other NPC) and the quest checks the "*NameNPC* has submitted".

**Comment by Rushor** [comment]
works for me https://i.imgur.com/tJnVpuZ.jpeg you were probally in a raid group

do: 

/script LeaveParty()

**Comment by Greywolf59** [comment]
I encountered this bug today. I went to Churn, used Death Grip (from max distance) and fought him. He turned green, said "ENOUGH" or something liek that and went back to his spot. However I did not get the credit for submision. 
Simmer worked fine, in his case I run right up to him, not risking the Death Grip could be a problem. Had to wait a lot of time before they become hostile again, probably like half an hour.
On the second try, without using Death Grip, both worked fine.
